docs/postgres.rst

```diff
@@ -117,6 +127,6 @@ run::
         --postgres-config homeserver-postgres.yaml
 
 Once that has completed, change the synapse config to point at the PostgreSQL
-database configuration file ``homeserver-postgres.yaml`` (i.e. rename it to 
-``homeserver.yaml``) and restart synapse. Synapse should now be running against
-PostgreSQL.
+database configuration file ``homeserver-postgres.yaml`` (for an example
```

Comment:
I think we should continue to tell people to rename it.

Reply:
why should it be called `homeserver-postgres.yaml` rather than just `homeserver.yaml`?

Reply:
I thought, the name is irrelevant. Just this instruction here should use only one name as example

Reply:
I think the difficult part here is, that you have to create that file yourself with the example above. It is irrelevant how you name it

Reply:
You're telling people to call it `homeserver-postgres.yaml`.

And while we're here, the user should already have written the file, so pointing them to example content now is unhelpful.